packages/astro/src/default/styles/variables.css

```diff
@@ -245,6 +245,10 @@
   --tk-elements-navCard-iconColor: var(--tk-elements-app-textColor);
   --tk-elements-navCard-iconColorHover: var(--tk-text-accent);
 
+  /* Webcontainers link */
+  --tk-elements-webcontainersLink-textColor: var(--tk-text-secondary);
+  --tk-elements-webcontainersLink-textColorHover: var(--tk-text-active);
```

Comment:
So what if we had primary and secondary links?

`editPageLink` could also maybe just a kind of primary link?

Reply:
Sure, I'll do that!